tests/ui/process/println-and-broken-pipe.run.stderr

```diff
@@ -0,0 +1,2 @@
+thread 'main' panicked at 'failed printing to stdout: Broken pipe (os error 32)', library/std/src/io/stdio.rs:1008:9
```

Comment:
We'll most likely want to filter out or wildcard `library/std/src/io/stdio.rs:1008:9` somehow. Before I figure out how to do that I wanted to check what CI thinks of my PR, and if there are any early code review comments.

Reply:
Actually, CR can wait until I have figured out what to do here.

@rustbot author

Reply:
you want to look into the `// normalize-stderr` directive (by looking at other tests that use it, i dont think there are any docs for compiletest lol)